source/Core/IOHandler.cpp

```diff
@@ -28,7 +28,9 @@
 #include "lldb/Core/State.h"
 #include "lldb/Core/StreamFile.h"
 #include "lldb/Core/ValueObjectRegister.h"
+#ifndef LLDB_DISABLE_LIBEDIT
```

Comment:
Does this file differ from the LLDB llvm.org SVN top of tree and the changes here are just syncing with top of tree SVN?